### DISCUSSION TITLE
Netflix key authentication does not work

### DISCUSSION THREAD

**Discussion by mpmeldal** [Koenkk/zigbee2mqtt#12875] [General]
repository.castagnait-2.0.0.zip
NFAuthenticationKey_Windows_1.2.0.0
Using the above repository on kodi and the key generator I do succesfully get into netflix using my account, but I am logged out immediately with reference to a log file, when I try to play a movie. I am running latest versions of libreelec/kodi on a rasberry pi 4B, and go through a VPN modem. Is this a new problem with netflix or is there a trick to getting it to work?
Mpm

**Comment by mpmeldal** [comment]
OK it works if I do not go through the VPN modem.
Should it not work with VPN?

**Reply by lux73** [reply]
what has this to do with z2m??
